syncd/syncd.cpp

```diff
@@ -40,6 +40,7 @@ void exit_and_notify(int status)
     exit(status);
 }
 
+#ifdef BRCMSAI
```

Comment:
This code breaks compilation as far as SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE is not supported anymore.